lib/http/endpoint.js

```diff
@@ -192,20 +192,28 @@ const endpointBase = ({ preprocessor = noop, before = noop, resultWriter, errorW
 // ENDPOINT FORMAT SPECIALIZATIONS
 
 ////////////////////////////////////////
```

Comment:
I think this line comment goes together with the line comment on the next line, so the two should be moved together. I think the two lines should come before `defaultResultWriter()`/`defaultErrorWriter()`/`defaultEndpoint()`, similar to how there are line comments above `odataJsonWriter()`/`odataXmlErrorWriter()`/`odataJsonEndpoint()`/etc.

Reply:
Could we just remove the comment completely?

Reply:
Sounds fine to me, I'm not attached to it. :+1: